spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,78 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:user) { create(:user, organisation: user_organisation) }
+  let(:user_organisation) { create(:organisation) }
```

Comment:
These two can be combined to one as the user factory creates an org with the user. See `factories/users.rb`.
So it becomes:
```
let(:user) { create(:user)
```